js/tab-manager.js

```diff
@@ -4,6 +4,18 @@ function restore(groupID) {
   });
 }
 
+function restoreToNewWindow(groupID) {
+  function onCreated() {
+    restore(groupID);
+  }
+
+  function onError(error) {
+    console.log(`Error: ${error}`);
```

Comment:
```suggestion
    console.debug(`Error: ${error}`);
```
Since that's how it's been for all the other cases.